dev/doc/release-process.md

```diff
@@ -126,9 +126,9 @@ in time.
 - [ ] Send an e-mail on Coqdev announcing that the tag has been put so that
   package managers can start preparing package updates (including a
   `coq-bignums` compatible version).
-- [ ] Ping **@erikmd** to update the Docker images in `coqorg/coq`
-  (requires `coq-bignums` in `extra-dev` for a beta / in `released`
-  for a final release).
+- [ ] When opening the corresponding PR for `coq` in the opam repository ([`coq/opam-coq-archive`](https://github.com/coq/opam-coq-archive) or [`ocaml/opam-repository`](https://github.com/ocaml/opam-repository)),
+  the package managers can Cc **@erikmd** to anticipate the Docker release in [`coqorg/coq`](https://hub.docker.com/r/coqorg/coq)
```

Comment:
This makes more sense now.

"anticipate" is not quite the right word here, though.  You might just say something like "to request that he prepare the necessary Docker files"

Reply:
Thanks.
typo here? → "that he prepares"

Reply:
Nevermind, "to request that he prepare" was OK (sorry for the noise)